codeGeneration.js

```diff
@@ -125,24 +151,26 @@ const buildBodyRule = fontData => {
 const buildStylesheet = (fontData, relativeFontPath) => {
   const root = postcss.root();
 
-  root.append([
-    buildFontFace(fontData, relativeFontPath),
-    buildBodyRule(fontData),
-    buildProperties(fontData),
-    buildFontVariationSettings(fontData)
-  ]);
+  root.append(buildFontFace(fontData, relativeFontPath));
+
+  if (fontData.data.axes.length) {
+    root.append(buildVarationVariables(fontData));
+    root.append(buildVariationStyles(fontData));
+  } else {
+    root.append(buildRegularStyles(fontData));
+  }
 
   return root;
 };
 
 const buildFontJs = fontData => {
-  return `export const fontName = "${fontData.name}";`;
+  return `fontNames.push("${fontData.name}");\n`;
```

Comment:
How does this work? It seems like `fontNames` is not declared, nor is it exported?

Reply:
It's not very clear from this code, but it ties into this: https://github.com/kabisa/specimen-skeleton/pull/58/files#diff-076df69e37321e03eb2edb6809afca0bR116

This way I don't have to keep track in this function if it's the first font, or the second/third/etc.

But I'm open to suggestions if this needs improvement!